python/lsst/ts/atwhitelight/config_schema.py

```diff
@@ -0,0 +1,190 @@
+__all__ = ["CONFIG_SCHEMA"]
+
+import yaml
+
+
+CONFIG_SCHEMA = yaml.safe_load(
+    """
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_atwhitelight/blob/master/schema/whitelight.yaml
+# title must end with one or more spaces followed by the schema version, which must begin with "v"
+title: Whitelight v2
+description: Schema for AT White Light configuration files
+type: object
+properties:
+  chiller:
+    description: Configuration for the ThermoTek chiller
+    type: object
+    properties:
+      host:
+        description: IP address of chiller.
+        type: string
+      port:
+        description: Port of chiller.
+        type: number
+      initial_temperature:
+        description: >-
+          The control temperature to specify when the CSC first connects to the chiller (C)
+          (When the CSC reconnects to the chiller it will command the most
+          recently specified control temperature.)
+          Must be in range [low_supply_temperature_warning, high_supply_temperature_warning], inclusive.
+        type: number
+      low_ambient_temperature_warning:
+        description: Threshold for publishing a warning for low ambient temperature (C)
+        type: number
+      high_ambient_temperature_warning:
+        description: Threshold for publishing a warning for high ambient temperature (C)
+        type: number
+      low_supply_temperature_warning:
+        description: Threshold for publishing a warning for low supply temperature (C)
+        type: number
+      high_supply_temperature_warning:
+        description: Threshold for publishing a warning for high supply temperature (C)
+        type: number
+      low_process_flow_warning:
+        description: Threshold for publishing a warning for low process flow (liters/min)
+        type: number
+        minimum: 0
+      low_ambient_temperature_alarm:
+        description: >-
+          Threshold for turning off the lamp and going to fault state
```

Comment:
in fault state does the chiller stop circulating?? Ideally there will be faults where we want the chiller to continue circulating....

Reply:
If the CSC goes to fault it leaves the chiller running until the cooldown period is over. Then, if the chiller is connected, the CSC shuts off the chiller.

Reply:
I reverted that change because the lamp controller does not reliably kill the lamp in an error state (one of its error states is "failed to extinguish the lamp"). So the answer is now: the chiller remains circulating until the user tells it to stop.